### PR TITLE
feat: add tests for filters component

### PR DESCRIPTION
**Pull request by sjaanus**

### CODE REVIEW

File: frontend/src/component/feature/FeatureToggleList/FeatureToggleFilters/FeatureToggleFilters.test.tsx [hunk 1]
```diff
@@ -39,21 +39,3 @@ test('should not render projects filters when less than two project', async () =
 
     expect(screen.queryByText('Projects')).not.toBeInTheDocument();
 });
-
-test('should keep filters order when adding a new filter', async () => {
```
Comment (sjaanus) [inline frontend/src/component/feature/FeatureToggleList/FeatureToggleFilters/FeatureToggleFilters.test.tsx:43]: Moving to appropriate file.

File: frontend/src/component/filter/Filters/Filters.test.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import { screen } from '@testing-library/react';
+import { render } from 'utils/testRenderer';
+import { FILTER_ITEM } from 'utils/testIds';
+import { Filters, IFilterItem } from './Filters';
+
+test('shoulder render all available filters', async () => {
+    const availableFilters: IFilterItem[] = [
```
Comment (kwasniew) [inline frontend/src/component/filter/Filters/Filters.test.tsx:7]: can we extract the fields that are not relevant into meaningful variables and only explicitly name the ones that affect the test?
e.g. const irrelevantOperators = ['irrelevant1', 'irrelevant2']
singularOperators: irrelevantsOperators

**Review by kwasniew — APPROVED** [review]

File: frontend/src/component/filter/Filters/Filters.test.tsx (hunk 2 above)
Comment (sjaanus) [reply frontend/src/component/filter/Filters/Filters.test.tsx:7]: Done

Comment (kwasniew) [reply frontend/src/component/filter/Filters/Filters.test.tsx:7]: same in the other test

Comment (sjaanus) [reply frontend/src/component/filter/Filters/Filters.test.tsx:7]: Damn, it seems violent, but I guess its for the greater good.

**Pull request merged**